poetry.lock

```diff
@@ -441,13 +441,13 @@ dev = ["freezegun (>=1.0,<2.0)", "pytest (>=6.0)", "pytest-cov"]
 
 [[package]]
 name = "backoff"
-version = "2.2.1"
+version = "1.11.1"
```

Comment:
Not necessarily a problem, I'll note for the record, backoff got downgraded

Reply:
I ran a full `poetry update`, which seems to have resolved this. Not sure why it downgraded...